plugins/services/src/js/containers/services/ServicesContainer.js

```diff
@@ -91,6 +92,15 @@ const SERVICE_FILTERS = new DSLFilterList([
 
 const notificationService = container.get(NotificationServiceType);
 
+function i18nTranslate(id, values) {
```

Comment:
does that mean that every containerModule should implement such a function and all marks would have to be wrapped in `i18nTranslate(i18nMark("meh"))` to not couple tests to having a i18n-containerModule loaded?

that does not feel elegant to me. maybe we can tackle that on a spike day.